src/index.html

```diff
@@ -1,14 +1,12 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title>Title</title>
-    <link
-      rel="stylesheet"
-      href="./styles/main.scss"
-    />
-  </head>
-  <body>
-    <script src="scripts/main.js"></script>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Title</title>
+  <link rel="stylesheet" href="./styles/main.scss" />
```

Comment:
Browsers cannot directly use `.scss` files. You need to compile `main.scss` to `main.css` and update this link to point to the compiled `.css` file.